catalog-core/src/main/java/uk/gov/justice/schema/catalog/CatalogUpdater.java

```diff
@@ -38,14 +42,20 @@ public void updateRawCatalog(final Map<String, String> schemaIdsToRawJsonSchemaC
             final Path updatedPath = Paths.get(format("%s/%s", basePath.toString(), path.toString()));
 
             try {
-                if(!updatedPath.toString().contains(CLASSPATH)){
+                if (!updatedPath.toString().contains(CLASSPATH)) {
                     final String schema = IOUtils.toString(updatedPath.toUri().toURL(), UTF_8);
                     try (final JsonReader reader = createReader(new StringReader(schema))) {
-                        final JsonObject jsonObject = reader.readObject();
+                        try {
+                            final JsonObject jsonObject = reader.readObject();
 
-                        if (jsonObject.containsKey("id")) {
-                            final String id = jsonObject.getString("id");
-                            schemaIdsToRawJsonSchemaCache.put(id, schema);
+                            if (jsonObject.containsKey("id")) {
+                                final String id = jsonObject.getString("id");
+                                schemaIdsToRawJsonSchemaCache.put(id, schema);
+                            }
+                        } catch (JsonParsingException e) {
+                            final String errorMessage = "Unable to parse Json file: " + updatedPath;
```

Comment:
we tend to use String.format() for concatenation, just to keep things consistent.